### ISSUE TITLE
GitElephant for Enterprise

### ISSUE THREAD

**Issue by GenieTim** [matteosister/GitElephant#161]
Dear fellow contributors

I would like to offer my services in connection with this repository/package as a lifter on [Tidelift](https://tidelift.com/), meaning e.g. enterprises would have the possiblity to sponsor certain issues. For this, I would like to ask you, @matteosister and @Imunhatep , whether you oppose this idea or, in case you may want to lift too, how the (potential) income would be split?

Best regards,

**Comment by phated** [comment]
I'm happy to answer any questions you might have about Tidelift, too!

**Comment by Imunhatep** [comment]
Well, this is @matteosister 's project.. Its up to him to decide.

From my point, this could be split by % of contributed code..

**Comment by matteosister** [comment]
I agree with @Imunhatep. Didn't have time to check what tidelift is, but it sounds like a good idea.

**Comment by phated** [comment]
We are currently unable to adjust the percents dynamically, but I can set them at a specific % to start and then we can revisit in the future.

I tried looking at the contributions graph but I'm actually not sure how that should be converted into the % split.

We also need @matteosister and @Imunhatep to sign up and complete our [Lifter Agreement](https://docs.tidelift.com/article/57-lifter-agreement). You can sign up by going to https://tidelift.com/lifter/search/packagist/cypresslab%2Fgitelephant and clicking "Lift your package".

**Comment by phated** [comment]
I've sent @matteosister the Lifter Agreement (just waiting for it to be completed), but haven't noticed @Imunhatep sign up yet.

If it is okay with all, I can get @GenieTim configured and then adjust things once everyone else has completed set up. Thoughts?
